datadog_checks_dev/datadog_checks/dev/tooling/configuration/consumers/model/model_consumer.py

```diff
@@ -12,7 +12,7 @@
 from datadog_checks.dev.tooling.configuration.consumers.model.model_info import ModelInfo
 from datadog_checks.dev.tooling.configuration.consumers.openapi_document import build_openapi_document
 
-PYTHON_VERSION = PythonVersion.PY_38
+PYTHON_VERSION = PythonVersion.PY_39
```

Comment:
It's time to bump this one then right? 

https://github.com/DataDog/integrations-core/blob/fd3586d689977fafdbcd84126127060c84eb24f5/ddev/pyproject.toml#L11